index.js

```diff
@@ -0,0 +1,132 @@
+// dependencies
+const inquirer = require("inquirer");
+const fs = require("fs");
+const createHTML = require("./src/html-creator.js");
+
+const Employee = require("./lib/employee.js");
```

Comment:
Your file names for classes should be capitalised to follow Class naming conventions